src/Main.hs

```diff
@@ -218,9 +220,10 @@ renderPage server baseUrl page = with html_ [lang_ "en"] $ do
             _topicLastUpdated t
           whenJust (listToMaybe $ _topicMessages t) $ \msg -> do
             ogpAttribute "article:published_time" $ ogpTimeFormat $ _messageTimestamp msg
-            -- FIXME: _messageContent is HTML, but we need plain text.
-            ogpAttribute "description" $ T.take 300 $ _messageContent msg
+            ogpAttribute "description" $ T.take 300 $ stripHtml $ _messageContent msg
```

Comment:
By the way, what's the reason for using a specific number (300) here?

Reply:
There's no max length for the description field, but 1-2 sentences are recommended and I think most popular services will truncate it after 200-300 characters. Technically, we could just use the full message text.